test/python/picongpu/quick/pypicongpu/runner.py

```diff
@@ -403,7 +403,7 @@ def test_init_sim_type(self):
         Runner(self.picmi_sim)
 
         for invalid_sim in [None, {}, 0, ""]:
-            with self.assertRaises(TypeError):
+            with self.assertRaises(typeguard.TypeCheckError):
```

Comment:
```suggestion
            with self.assertRaises(TypeError):
```
As opposed to the others, this is indeed a manually raised `TypeError` from [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/7edbba8d21172f97cad0e6bd84fdb93098773844/lib/python/picongpu/pypicongpu/runner.py#L194). Not sure if that's desired but at least this should fix the CI.